src/js/jsx/sections/export/ExportList.jsx

```diff
@@ -188,28 +195,36 @@ define(function (require, exports, module) {
         render: function () {
             var document = this.props.document,
                 layers = this.props.layers,
-                layer = layers && layers.size > 0 && layers.first(), // simplification
                 documentExports = this.props.documentExports,
-                exportsList,
+                keyprefix = layers && collection.pluck(layers, "key").join("-") || document.id,
+                assetGroups,
                 exportComponents;
 
-            if (layer) {
-                exportsList = documentExports && documentExports.layerExportsMap.get(layer.id);
-            } else {
-                exportsList = documentExports && documentExports.rootExports;
+            if (documentExports) {
+                if (layers && layers.size > 0) {
+                    var allExports = layers.map(function (layer) {
+                        return documentExports.getLayerExports(layer.id);
+                    });
+                    
+                    assetGroups = collection.zip(allExports);
+                } else {
+                    assetGroups = collection.zip(Immutable.List.of(documentExports.rootExports));
+                }
             }
 
-            if (!exportsList || exportsList.size < 1) {
+            if (!assetGroups || assetGroups.size < 1) {
```

Comment:
Wait, I thought that `.zip` returned a Seq which does not have a `size` property?